server/hydrogen-render/3-render-hydrogen-to-string-unsafe.js

```diff
@@ -74,6 +76,12 @@ async function _renderHydrogenToStringUnsafe({ fromTimestamp, roomData, events,
   // Make sure `webcrypto` exists since it was only introduced in Node.js v17
   assert(crypto.webcrypto);
   vmContext.global.crypto = crypto.webcrypto;
+  // LocalStorage stuff
+  vmContext.global.localStorage = new MockLocalStorage();
+  // IndexDB stuff
+  vmContext.global.indexedDB = new IDBFactory();
+  vmContext.global.IDBFactory = IDBFactory;
+  vmContext.global.IDBKeyRange = IDBKeyRange;
```

Comment:
More dependencies is annoying. Wish Hydrogen could work without storage.

`fake-indexeddb` is at least what Hydrogen is using as well.